end2end-test-examples/echo-client/README.md

```diff
@@ -47,6 +47,8 @@ Per sec Payload = 0.07 MB (exact amount of KB = 10000)
 
 `--interval`: Interval in ms between gRPC calls. For finite number of requests the default value is 0, 1000 ms otherwise.
 
+`--dropChannel`: Re-create the channel before each request. Not applicable with --async.
```

Comment:
why not call this param: "recreateChannelAlways" or "recreateChannelOnEveryRequest" or something like that. toolong a name?

Reply:
Remade into recreateChannelSeconds argument. This way we can control how often we want to recreate the channel.